packages/cli-kit/src/public/node/themes/api.test.ts

```diff
@@ -35,6 +35,12 @@ vi.stubGlobal('fetch', vi.fn())
 const session = {token: 'token', storeFqdn: 'my-shop.myshopify.com', refresh: async () => {}}
 const themeAccessSession = {...session, token: 'shptka_token'}
 const sessions = {CLI: session, 'Theme Access': themeAccessSession}
+const expectedApiOptions = expect.objectContaining({
```

Comment:
nit: let's keep the naming for this consistent with the actual argument naming (e.g. requestBehaviourOptions)? LMK If I'm missing something though